### PR TITLE
[stable/redis] add `volumeMount` `subPath` feature.

### PR DESCRIPTION
**Pull request by Antiarchitect**
Ability to use subdirectories in PV. Helps with Dev under minikube and cases when one PV for multiple services is used.

### CODE REVIEW

File: stable/redis/values.yaml [hunk 1]
```diff
@@ -44,6 +44,9 @@ persistence:
   ## Redis images.
   path: /bitnami
 
+  ## The subdirectory of the volume to mount to, useful in dev environments and one PV for multiple services.
+  supPath: ""
```
Comment (sameersbn) [inline stable/redis/values.yaml]: you have a typo here

**Review by sameersbn — CHANGES_REQUESTED** [review]

**Comment by Antiarchitect** [comment]
@sameersbn Thx, fixed.

**Pull request merged**